### PR TITLE
Split client-js into core-client-js and client-js

### PR DESCRIPTION
**Pull request by nerfZael**

### CODE REVIEW

File: packages/js/core-client/jest.rs.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
```
Comment (cbrzn) [inline packages/js/core-client/jest.rs.config.js]: I think this file can be removed, since the tests are in `client`

**Review by cbrzn — COMMENTED** [review]
cool! do you think it would make sense to move some tests to `core-client` package too? to test at least the main functionalities (like get env or get implementations)?

File: packages/js/core-client/jest.rs.config.js (hunk 1 above)
Comment (nerfZael) [reply packages/js/core-client/jest.rs.config.js]: I left this so we can add tests later on

**Comment by nerfZael** [comment]
> cool! do you think it would make sense to move some tests to `core-client` package too? to test at least the main functionalities (like get env or get implementations)?

Yea, I think a lot of tests could probably be moved
Lets do that in another PR though

**Pull request merged**